src/impact-dashboard/CurveChart.tsx

```diff
@@ -61,7 +65,8 @@ const Tooltip: React.FC<TooltipProps> = ({
         <TooltipDatum>
           <DateMMMMdyyyy date={displayDate} />
         </TooltipDatum>
-        <TooltipDatum>People: {formatThousands(count)}</TooltipDatum>
+        <TooltipDatum>Inmates: {formatThousands(countInmates)}</TooltipDatum>
```

Comment:
@sychang can we find another word to use here than _Inmates_? Maybe just _Incarcerated_? I like to avoid the former term where possible and we've successfully avoided it in this project to date.

Reply:
Yes, "Incarcerated" please!